tests/many_to_one/tests.py

```diff
@@ -576,6 +577,15 @@ def test_fk_to_bigautofield(self):
         District.objects.create(city=ny, name='Brooklyn')
         District.objects.create(city=ny, name='Manhattan')
 
+    def test_fk_to_smallautofield(self):
```

Comment:
This and `test_fk_to_bigautofield` above are a funny pair of tests. (`assert did_not_blow_up`... :)